tools/mason/mason.chpl

```diff
@@ -132,5 +182,5 @@ proc masonClean(args) {
 
 
 proc printVersion() {
-  writeln('mason 0.1.2');
+  writeln("mason 0.1.2");
```

Comment:
Should we be bumping the minor mason version number for the upcoming release?

Reply:
I think you are right. I seem to recall seeing a note about this somewhere, but now I can't find it.

Reply:
I will leave this as-is for now and possibly increment it when I complete cray/chapel-private#2454